beacon-chain/blockchain/service.go

```diff
@@ -226,6 +196,43 @@ func (c *ChainService) updateHead(slot uint64) {
 	c.candidateCrystallizedState = nilCrystallizedState
 }
 
+func (c *ChainService) getParentSlot(block *types.Block) (uint64, error) {
```

Comment:
Should this be under `core.go`?

Reply:
Ah, yeah, this is going to go away after the switch to Bolt, but until then this belongs in `core.go`